src/layouts/EditPage/EditPage.tsx

```diff
@@ -106,10 +113,30 @@ export const EditPage = () => {
     <EditorContextProvider editor={editor}>
       <EditorModalContextProvider
         showModal={(modalType) => {
-          setMediaType(modalType)
-          onMediaModalOpen()
+          if (modalType === "hyperlink") {
+            onHyperlinkModalOpen()
+          } else {
+            setMediaType(modalType)
+            onMediaModalOpen()
+          }
         }}
       >
+        {isHyperlinkModalOpen && (
+          <HyperlinkModal
+            initialText=""
+            onSave={(text, href) => {
+              editor
+                .chain()
+                .focus()
+                .insertContent(
+                  `<a target="_blank" rel="noopener noreferrer nofollow" href=${href}>${text}</a>`
```

Comment:
are we missing "" around the href?

Reply:
tiptap adds it back but i'll just add back here for safety/consistency